app/models/remote_theme.rb

```diff
@@ -6,6 +6,9 @@ class RemoteTheme < ActiveRecord::Base
 
   ALLOWED_FIELDS = %w{scss embedded_scss head_tag header after_header body_tag footer}
 
+  GITHUB_REGEXP = /^(http|https):\/\/github\.com\//
```

Comment:
This Regexp can be a tiny bit shorter: `/^https?:\/\/github\.com\//`  ;)

Reply:
Thanks! :D